src/sentry/static/sentry/app/views/organizationDiscover/discover.jsx

```diff
@@ -46,6 +46,31 @@ export default class OrganizationDiscover extends React.Component {
     );
   };
 
+  getOrderbyOptions = () => {
```

Comment:
Would this be better in query builder?

Reply:
I'd initially put some similar stuff to this in querybuilder, but have since removed it and kept that exclusively for managing query state and converting between internal and external formats.